Add qos params for topos ft2-64, lt2-p32o64, lt2-o128

rick-arista opened the pull request:
<!--
Please make sure you've read and understood our contributing guidelines;
https://github.com/sonic-net/SONiC/blob/gh-pages/CONTRIBUTING.md

Please provide following information to help code review process a bit easier:
-->
### Description of PR
<!--
- Please include a summary of the change and which issue is fixed.
- Please also include relevant motivation and context. Where should reviewer start? background context?
- List any dependencies that are required for this change.
-->

Summary:
Adds qos test params for:
 - ft2-64
 - lt2-p32o64
 - lt2-o128

### Type of change

<!--
- Fill x for your type of change.
- e.g.
- [x] Bug fix
-->

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] New Test case
    - [ ] Skipped for non-supported platforms
- [ ] Test case improvement


### Back port request
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [ ] 202405
- [ ] 202411
- [ ] 202505

### Approach
Manually ran `qos/test_qos_sai.py` tests, output:

```
topo-ft2-64:
qos/test_qos_sai.py::TestQosSai::testParameter[single_asic] PASSED                                                                                                                                                                                                                
qos/test_qos_sai.py::TestQosSai::testQosSaiPfcXoffLimit[single_asic-xoff_1] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiPfcXoffLimit[single_asic-xoff_2] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiPfcXonLimit[single_asic-xon_1] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiPfcXonLimit[single_asic-xon_2] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiLossyQueue[single_asic] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiDscpQueueMapping[single_asic] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiDwrr[single_asic] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiPgSharedWatermark[single_asic-wm_pg_shared_lossless] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiPgSharedWatermark[single_asic-wm_pg_shared_lossy] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiPgHeadroomWatermark[single_asic] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiQSharedWatermark[single_asic-wm_q_shared_lossless] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiQSharedWatermark[single_asic-wm_q_shared_lossy] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiDwrrWeightChange[single_asic] PASSED
======== 14 passed, 266 skipped, 1 warning in 2989.80s (0:49:49) =====================================

topo-lt2-p32o64:
qos/test_qos_sai.py::TestQosSai::testParameter[single_asic] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiPfcXoffLimit[single_asic-xoff_1] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiPfcXoffLimit[single_asic-xoff_2] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiPfcXonLimit[single_asic-xon_1] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiPfcXonLimit[single_asic-xon_2] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiLossyQueue[single_asic] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiDscpQueueMapping[single_asic] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiDwrr[single_asic] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiPgSharedWatermark[single_asic-wm_pg_shared_lossless] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiPgSharedWatermark[single_asic-wm_pg_shared_lossy] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiPgHeadroomWatermark[single_asic] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiQSharedWatermark[single_asic-wm_q_shared_lossless] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiQSharedWatermark[single_asic-wm_q_shared_lossy] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiDwrrWeightChange[single_asic] PASSED
======== 14 passed, 266 skipped, 1 warning in 3483.55s (0:58:03) =====================================

topo-lt2-o128:
qos/test_qos_sai.py::TestQosSai::testParameter[single_asic] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiPfcXoffLimit[single_asic-xoff_1] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiPfcXoffLimit[single_asic-xoff_2] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiPfcXonLimit[single_asic-xon_1] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiPfcXonLimit[single_asic-xon_2] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiLossyQueue[single_asic] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiDscpQueueMapping[single_asic] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiDwrr[single_asic] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiPgSharedWatermark[single_asic-wm_pg_shared_lossless] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiPgSharedWatermark[single_asic-wm_pg_shared_lossy] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiPgHeadroomWatermark[single_asic] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiQSharedWatermark[single_asic-wm_q_shared_lossless] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiQSharedWatermark[single_asic-wm_q_shared_lossy] PASSED
qos/test_qos_sai.py::TestQosSai::testQosSaiDwrrWeightChange[single_asic] PASSED
======== 14 passed, 266 skipped, 1 warning in 3559.45s (0:59:19) =====================================
```

mssonicbld commented:
/azp run

rick-arista commented:
Hi @XuChen-MSFT and @bingwang-ms. I've updated this PR with topo-lt2-p32o64 params. I'll add the topo_ft2-p64 params shortly. These were all manually tested and I dont recall seeing any key errors.

rick-arista commented:
@bingwang-ms A separate backport PR has been opened against the branch directly

bingwang-ms commented:
Hi @rick-arista, would you mind fill the PR template properly and also include a test result so that we can have this PR merged?

mssonicbld commented:
/azp run

rick-arista commented:
@bingwang-ms I've added the ft2-64 params. I'll run the qos tests for these topos today and update the PR description with the results. Will also fix the PR title and template

mssonicbld commented:
/azp run

rick-arista commented:
@bingwang-ms @XuChen-MSFT 

I've updated the params for these 3 topos, and fixed the title and description of this PR. I've also addressed the comments in the associated PR for the for the test fixes that needed to be backported.

bingwang-ms reviewed (approved):
LGTM
